tests/copilot_helper/test_command_copilot.py

```diff
@@ -201,6 +203,68 @@ def test_make_addons_success(
             len(actual_files) == len(all_expected_files) + 3
         ), "The actual filecount should be expected files plus 2 initial manifest.yml and override files"
 
+    @freeze_time("2023-08-22 16:00:00")
+    @patch(
+        "dbt_copilot_helper.utils.versioning.running_as_installed_package",
+        new=Mock(return_value=True),
```

Comment:
```suggestion
        new=Mock(return_value=False),
```

And you can remove `validate_version` stuff from this test, it's covered in the one above.